### PR TITLE
[Release] Update versions for Release 11.6.0

### PR DESCRIPTION
**Pull request by andrewheard**
Update versions to 11.6.0 in preparation for the next release.

#no-changelog

### CODE REVIEW

File: FirebaseAnalytics.podspec [hunk 1]
```diff
@@ -26,7 +26,7 @@ Pod::Spec.new do |s|
     s.libraries  = 'c++', 'sqlite3', 'z'
     s.frameworks = 'StoreKit'
 
-    s.dependency 'FirebaseCore', '11.5'
+    s.dependency 'FirebaseCore', '11.6.0'
```
Comment (paulb777) [inline FirebaseAnalytics.podspec:29]: This should be "~> 11.6"

File: FirebaseFirestore.podspec [hunk 2]
```diff
@@ -35,9 +35,9 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
     "#{s.module_name}_Privacy" => 'Firestore/Swift/Source/Resources/PrivacyInfo.xcprivacy'
   }
 
-  s.dependency 'FirebaseCore', '11.5'
-  s.dependency 'FirebaseCoreExtension', '11.5'
-  s.dependency 'FirebaseFirestoreInternal', '11.5.0'
+  s.dependency 'FirebaseCore', '11.6.0'
+  s.dependency 'FirebaseCoreExtension', '11.6.0'
```
Comment (paulb777) [inline FirebaseFirestore.podspec:39]: This also needs. a ~

File: FirebaseCore.podspec [hunk 3]
```diff
@@ -53,7 +53,7 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   # Remember to also update version in `cmake/external/GoogleUtilities.cmake`
   s.dependency 'GoogleUtilities/Environment', '~> 8.0'
   s.dependency 'GoogleUtilities/Logger', '~> 8.0'
-  s.dependency 'FirebaseCoreInternal', '11.5'
+  s.dependency 'FirebaseCoreInternal', '11.6.0'
```
Comment (paulb777) [inline FirebaseCore.podspec:56]: This too.

**Review by paulb777 — APPROVED** [review]
OK to merge and fix the FirebaseCore deps in a follow up - if the ci failures are separately addressed.

File: FirebaseAnalytics.podspec (hunk 1 above)
Comment (andrewheard) [reply FirebaseAnalytics.podspec:29]: Should all of these (https://github.com/search?q=repo%3Afirebase%2Ffirebase-ios-sdk+%22%2C+%2711.5.0%27%22&type=code) be `'~> 11.6'`?

**Comment by andrewheard** [comment]
> OK to merge and fix the FirebaseCore deps in a follow up - if the ci failures are separately addressed.

@paulb777 If it's alright, I'll hold off on merging until tomorrow. Will make sure that CI passes on this PR and then double check our versioning strategy.

**Comment by andrewheard** [comment]
> SGTM. Note there may be some nightly failures because the new release tag exists, but is not yet merged

In that case it probably is easier to merge. I can always touch all the podspecs to cause the same checks to run again tomorrow.

File: FirebaseAnalytics.podspec (hunk 1 above)
Comment (andrewheard) [reply FirebaseAnalytics.podspec:29]: Will address in a follow-up PR.

**Pull request merged**